app/models/internal/impl/DefaultAlertEvaluationResult.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.internal.impl;
+
+import com.arpnetworking.commons.builder.OvalBuilder;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import models.internal.AlertEvaluationResult;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Default implementation for an {@code AlertEvaluationResult}.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+@JsonDeserialize(builder = DefaultAlertEvaluationResult.Builder.class)
```

Comment:
I would think this is handled by the standard object mapper config but without this Jackson complains that it can't deserialize `ImmutableList` (since it's performing it through reflection and not through the setter below.

Reply:
Where is this deserialized?